openslides/users/static/js/users/site.js

```diff
@@ -707,9 +711,17 @@ angular.module('OpenSlidesApp.users.site', ['OpenSlidesApp.users'])
         $scope.pageChanged = function() {
             $scope.limitBegin = ($scope.currentPage - 1) * $scope.itemsPerPage;
         };
+        $scope.duplicateActions = [
+            'keep original',
+            'override new',
+            'create duplicate'
```

Comment:
Default should be the action "override new" because the most important use case whiche shows duplicates comes if someone want to overide existing users. So please change it.

Wish: Changing all duplicates to an other action with a global select could be helpful.